.taskcluster.yml

```diff
@@ -0,0 +1,30 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.closed
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+          - push
+          - release
+    payload:
+      maxRunTime: 3600
+      image: node
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} repo && cd repo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} && npm
+          install . && npm test
+    metadata:
+      name: Run tests for taskcluster-login in Taskcluster
+      description: create .taskcluster.yml file
```

Comment:
Let's take what you have for `name` and make it the `description`.  The `name` can be `taskcluster-login tests`